src/main/java/JerChatBot.java

```diff
@@ -0,0 +1,243 @@
+import datahandling.FileHandling;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import exception.EmptyDescriptionException;
+import exception.UnknownCommandException;
+import exception.InvalidTaskNumberException;
+import exception.TaskListFullException;
+import task.Task;
+import task.Event;
+import task.Deadline;
+import task.ToDo;
+import task.UserManual;
+
+public class JerChatBot {
+    private static final String FILE_PATH = "./data/data.txt";
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+    private static final FileHandling datahandling = new FileHandling(FILE_PATH);
+
+    private static final String TODO = "todo";
+    private static final String DEADLINE = "deadline";
+    private static final String EVENT = "event";
+
+    public static void main(String[] args) {
+        tasks.addAll(datahandling.loadTasksFromFile());
+        Scanner scanner = new Scanner(System.in);
+        printWelcomeMessage();
+        UserManual.printUserManual();
+
+        while (true) {
+            String input = scanner.nextLine().trim();
+
+            if (input.isEmpty()) {
+                printEmptyInputMessage();
+                continue;
+            }
+
+            String[] commands = input.split(" ", 2);
+            String command = commands[0];
+
+            try {
+                switch (command) {
+                case "bye":
```

Comment:
Perhaps there is more to be done for magic strings and integers for the entire codebase?